apiary.apib

```diff
@@ -59,19 +58,91 @@ Any response that does not provide a ```200 OK``` response code should be treate
 
 ### GET
 
-+ Response 200 (application/json)
++ Response 200 (application/json; charset=utf-8)
     + Headers
-    
+
             X-Dude-Api-Limit: Unlimited or Number
             X-Dude-Api-Remaining: Unlimited or Number
             X-Dude-Api-Server: Api Server ID
-            
+
     + Attributes (Heartbeat)
 
+
+# Group Inventory Item
+
+## Inventory Item Collection [/accounts/{accountId}/inventoryitems{?page,pagesize,orderby}]
+
++ Parameters
+    + accountId: 2 (required, number) - {INT} ID of the account in the form of an integer
+    + page: 1 (number, optional) - {INT} The page to request from the collection.
+    + pagesize: 1000 (number, optional) - {INT} The number of records to load in a page from the collection.
+    + orderby: Id desc (string, optional) - A string describing the ordering to perform on the collection.
+
+### GET
+
++ Response 200 (application/json; charset=utf-8)
+    + Attributes (Base Collection Response)
+        + `data` (array[Inventory Item Update], required)
+
+### POST
+
++ Request
+    + Attributes (Inventory Item Data)
+
++ Response 201 (application/json; charset=utf-8)
+    + Attributes (Inventory Item Update)
+
+## Inventory Item Detail [/accounts/{accountId}/inventoryitems/{id}]
+
++ Parameters
+    + accountId: 2 (number, required) - {INT} ID of the account in the form of an integer
+    + id: 1 (number, required) - {INT} The ID of the Inventory Item
+
+### GET
+
++ Response 200 (application/json; charset=utf-8)
+    + Attributes (Inventory Item Update)
+
+### PUT
+
++ Request
+    + Attributes (Inventory Item Update)
+
++ Response 200 (application/json; charset=utf-8)
+    + Attributes (Inventory Item Data)
+
+### DELETE
+
++ Response 204 (application/json; charset=utf-8)
+
 # Data Structures
```

Comment:
All Data structures should have a description and a separate section for parameters. see standards documentation: https://readme.dudesoln.com/display/EA/Apiary+Documentation+Standards